core/src/main/java/c/org/rajawali3d/logging/LoggingComponent.java

```diff
@@ -0,0 +1,383 @@
+package c.org.rajawali3d.logging;
+
+import c.org.rajawali3d.annotations.DebugOnly;
+import org.rajawali3d.BuildConfig;
+import timber.log.Timber;
+
+/**
+ * Instance-specific debug-mode and release-mode logging and debug utilities
+ *
+ * @author Randy Picolet
+ */
+
+public abstract class LoggingComponent {
```

Comment:
Rather than rolling our own here, I think it makes the most sense to utilize `Timber`. It has already been added as a dependency.

Reply:
Um, I did use Timber...   The purpose of this class is to add capabilities beyond static logging (more/better context for each message, plus asserts and configurable method tracing), but underneath everything there are the two Timber trees - DebugLog and ReleaseLog, whichever one is planted will handle all the messages generated in LoggingComponent...  I chose to wrap Timber rather than have Timber dependencies spread through the engine.  If we hit any problems with Timber, we can then replace it very easily.  Am i missing your concern?

Reply:
No I missed the usage in all of the PR. My appologies.

Reply:
Not a problem.  There is a lot to absorb, all feedback is great, and I have no doubt I will take my turns with missing stuff.